dashboard/of-cloud-dashboard/handler.js

```diff
@@ -62,11 +84,14 @@ module.exports = (event, context) => {
     headers['Content-Type'] = 'text/html';
     content = fs.readFileSync(`${__dirname}/dist/index.html`).toString();
```

Comment:
Please raise an Issue to move `fs.readFileSync` to `fs.readFile` everywhere in this file.

Reply:
Done